bin/ci

```diff
@@ -6,18 +6,15 @@ set -e
 echo 💡  Running tests...
 
 mix test
-npm install
-npm test
 
 cd ./examples/example-phx-1_2
 mix deps.get
 mix test
 
 cd ../example-phx-1_3
 mix deps.get
+nohup chromedriver &
 mix test
+killall chromedriver
```

Comment:
Loads up chromedriver in a background process, runs the feature tests, and kills any chromedriver processes afterward.

I'm purposely doing it this way rather than saving the pid. If there are other chromedriver processes hanging around, I want them dead.